src/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
'uuid' should be listed in the project's dependencies. Run 'npm i -S uuid' to add it